go.mod

```diff
@@ -1 +1,3 @@
 module github.com/nttcom/eclcloud
+
+go 1.13
```

Comment:
It has been added automatically. I think there is no negative impact. (I will erase it if necessary. )